### PR TITLE
Add v1.4.0 of badhaloninja.UserJoinLeaveNotifications

### PR DESCRIPTION
**Pull request by badhaloninja**

### CODE REVIEW

**Review by ljoonal — CHANGES_REQUESTED** [review]
> ❯ ./review.sh me.badhaloninja.UserJoinLeaveNotifications 1.4.0
SHA256 sums don't match!
Expected: 0837ab104cb5dc55eec21c27dc4d84cb886d5eb1acdc3a6034a598c0be288cdf
Actual:   c5029d87001b85d4cc62f120da94afe7e895ebafed04f3ebc0efd8e388c76d61

File: manifest.json [hunk 1]
```diff
@@ -4133,6 +4133,17 @@
                 }
             },
             "versions": {
+                "1.4.0": {
+                    "changelog": "Add a config to enable friend links on the notifications - EIA485",
+                    "releaseUrl": "https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/tag/v1.3.0",
+                    "artifacts": [
+                        {
+                            "url": "https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/download/v1.3.0/UserJoinLeaveNotifications.dll",
```
Comment (EIA485) [inline manifest.json:4141]: looks like the reason for the checksums not matching is this link being incorrect

Comment (EIA485) [reply manifest.json:4141]: https://github.com/badhaloninja/UserJoinLeaveNotifications/releases/download/v1.4.0/UserJoinLeaveNotifications.dll returns a file with the correct checksum

**Comment by badhaloninja** [comment]
Oops, should be fixed now

**Review by XDelta — APPROVED** [review]

**Comment by XDelta** [comment]
> > ❯ ./review.sh me.badhaloninja.UserJoinLeaveNotifications 1.4.0
> > SHA256 sums don't match!
> > Expected: 0837ab104cb5dc55eec21c27dc4d84cb886d5eb1acdc3a6034a598c0be288cdf
> > Actual:   c5029d87001b85d4cc62f120da94afe7e895ebafed04f3ebc0efd8e388c76d61

Github being weird and I can't seem to mark this as resolved.

**Comment by EIA485** [comment]
lj has to be the one to remove it

**Review by ljoonal — APPROVED** [review]

**Pull request merged**